partials/header.hbs

```diff
@@ -28,21 +27,21 @@
         </div>
     {{/if}}
     <div class="header-right hidden-xs hidden-sm hidden-md">
-        {{#if @custom.content_api_key_for_search}}
-            <button class="button-icon menu-item menu-item-button js-modal" data-modal="search" aria-label="Search">
-                <svg class="icon">
-                    <use xlink:href="#magnify"></use>
-                </svg>
-            </button>
-        {{/if}}
-        {{navigation}}
+        <button class="button-icon js-modal" data-modal="search" aria-label="Search">
+            <svg class="icon">
+                <use xlink:href="#magnify"></use>
+            </svg>
+        </button>
+        <a class="menu-item menu-item-impact">
+            {{navigation}}
+        </a>
     </div>
 </header>
 
 {{#is "post"}}
     {{#post}}
         <header class="sticky">
-            <div class="sticky-title">{{title}}</div>
+            <div class="sticky-title">{{tite}}</div>
```

Comment:
Sorry I don't really get this part

Reply:
I think this may have created another bug where the title isn't shown, so need to change back to `{{title}}` ?

![image](https://user-images.githubusercontent.com/18213722/204695668-6ed23db1-b430-40fe-b10b-14abd5e4bfb5.png)